src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -931,9 +949,7 @@ private bool CheckPropertyValueKind(SyntaxNode node, BoundExpression expr, BindV
 
             // Addendum: Assignment is also allowed for get-only autoprops in their constructor
 
-            BoundExpression receiver;
-            SyntaxNode propertySyntax;
-            var propertySymbol = GetPropertySymbol(expr, out receiver, out propertySyntax);
+            var propertySymbol = GetPropertySymbol(expr, out BoundExpression receiver, out SyntaxNode propertySyntax);
```

Comment:
>out BoundExpression receiver, out SyntaxNode propertySyntax) [](start = 57, length = 60)

I think refactorings like this negatively impact readability of the code. Was there something wrong with the code? Does it violate style, analyzer complained? Was this change necessary to accomplish the task?